packages/shared-ux/card/no_data/mocks/src/jest.ts

```diff
@@ -60,6 +63,11 @@ export const getKibanaDependenciesMock = (
         },
       },
     },
-    getRedirectAppLinksKibanaDependenciesMock()
+    getRedirectAppLinksKibanaDependenciesMock(),
+    {
+      isMergeableObject: isPlainObject,
```

Comment:
What is this change?

Reply:
`deepmerge` merges objects, so `jest.fn()` objects were also getting merged, which meant they were no longer mocked functions.  `isMergeableObject` allows one to create a function to test if the object should be merged or not.  In this case, I'm saying `deepmerge` should only merge the object if it passes `lodash.isPlainObject`.